src/docs/src/config/csrt.rst

```diff
@@ -0,0 +1,587 @@
+
+    .. config:option:: query_cardinality_limit :: Maximum quantity of rows to allow in CSRT query/http requests.
+
+        Limit the quantity of rows that can be loaded in an http query.::
+
+            [csrt]
+            query_cardinality_limit = 10000
+
+.. _csrt-logger-matcher-configuration:
+
+CSRT Logger Matcher Configuration
+=================================
+
+There are currently eight builtin default logger matchers designed to make it
+easy to do filtering on heavy resource usage inducing and long running
+requests. These are designed as a simple baseline of useful matchers, declared
+in a manner amenable to `default.ini` based constructs. More expressive matcher
+declarations are being explored, and matchers of arbitrary complexity can be
+registered directly through `remsh`. The default matchers are all designed around
+an integer config Threshold that triggers on a specific field, eg docs read, or
+on a delta of fields for long requests and changes requests that process many
+rows but return few.
+
+The current default matchers are:
+
+  * `all_coordinators`: match all Coordinators handling HTTP requests
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/all_coordinators>` | none
+
+  * `all_rpc_workers`: match all RPC Worker handling internal requests
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/all_rpc_workers>` | none
+
+  * `docs_read`: match all requests reading more than N docs
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/docs_read>` | :config:option:`Threshold <csrt_logger.matchers_threshold/docs_read>`
+
+  * `rows_read`: match all requests reading more than N rows
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/rows_read>` | :config:option:`Threshold <csrt_logger.matchers_threshold/rows_read>`
+
+  * `docs_written`: match all requests writing more than N docs
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/docs_written>` | :config:option:`Threshold <csrt_logger.matchers_threshold/docs_written>`
+
+  * `ioq_calls`: match all requests inducing more than N ioq_calls
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/ioq_calls>` | :config:option:`Threshold <csrt_logger.matchers_threshold/ioq_calls>`
+
+  * `long_reqs`: match all requests lasting more than N milliseconds
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/long_reqs>` | :config:option:`Threshold <csrt_logger.matchers_threshold/long_reqs>`
+
+  * `changes_processed`: match all changes requests that returned at least N rows
+    less than was necessarily loaded to complete the request (eg find heavy
+    filtered changes requests reading many rows but returning few).
+
+    * :config:option:`Enable <csrt_logger.matchers_enabled/changes_processed>` | :config:option:`Threshold <csrt_logger.matchers_threshold/changes_processed>`
+
+Each of the default matchers has an enablement setting in
+ref:`csrt-logger-matcher-configuration-enablement` for toggling enablement of
```

Comment:
May need another `:` in front as it doesn't render as a link reference currently.

<img width="574" height="106" alt="Screenshot 2025-09-02 at 10 11 07 PM" src="https://github.com/user-attachments/assets/7deaeee1-2dd3-42a3-b972-248f4c4f3507" />